cloud_cleaner/resources/server.py

```diff
@@ -111,6 +127,73 @@ def __right_name(self, target):
                self.__name.match(target.name)
 
     def __right_age(self, target):
-        system_age = datetime.strptime(target.created, DATE_FORMAT)
+        system_age = datetime.strptime(target.launched_at, DATE_FORMAT)
         system_age = system_age.replace(tzinfo=utc)
         return self._now > (system_age + self._interval)
+
+    def __set_flagged(self):
```

Comment:
We had discussed the idea of keeping state, but I thought we opted to go with the simpler approach of just having the user call with a dates. Then they can do something like, "Dry-run for >2 weeks with email, real run for >3 weeks with email and deletion." That way we don't need to keep the state between runs.

Reply:
I remember you mentioning that, I had changed it to require fewer calls(originally I had written it needing 2 calls of cloud-cleaner), but now I understand more what you're saying. I think that is a good idea, I was originally apprehensive because I was afraid of specific edge cases(using your example, cloud cleaner being called when a server is 1 week 6 days old, then being called when the same server is 3 weeks old, resulting in the server being deleted without a warning email ever sent), but thinking about it, I think most can be solved by just calling cloud cleaner regularly, which I believe the plan will be anyway. Also this change would reduce the number of additional files needed to use cloud-cleaner, which feels like a good thing.